src/Standards/AcquiaPHPStrict/ruleset.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset -->
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="../../../vendor/squizlabs/php_codesniffer/phpcs.xsd"
+         name="AcquiaPHPStrict"
+>
+
+  <description>Acquia's strict PHP coding standards.</description>
+
+  <rule ref="AcquiaPHP"/>
+
+  <!-- Drupal sniffs -->
+  <rule ref="Drupal.Commenting.InlineComment"/>
+
+  <!-- SlevomatCodingStandard sniffs -->
+  <rule ref="SlevomatCodingStandard.Arrays.AlphabeticallySortedByKeys" />
+  <rule ref="SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation"/>
+  <rule ref="SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement"/>
+  <rule ref="SlevomatCodingStandard.Arrays.SingleLineArrayWhitespace"/>
+  <rule ref="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+  <rule ref="SlevomatCodingStandard.Classes.RequireConstructorPropertyPromotion" />
+  <rule ref="SlevomatCodingStandard.Commenting.DeprecatedAnnotationDeclaration" />
+  <rule ref="SlevomatCodingStandard.Commenting.DisallowCommentAfterCode" />
+  <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing" />
+  <rule ref="SlevomatCodingStandard.Commenting.EmptyComment" />
+  <rule ref="SlevomatCodingStandard.Commenting.ForbiddenAnnotations">
+    <properties>
+      <property name="forbiddenAnnotations" type="array" value="@author,@created,@version,@package,@copyright,@license,@throws" />
```

Comment:
I use `@throws` extensively because it helps PHPStan and PhpStorm identify un-caught exceptions. I would "throw" a fit (pun intended? 😉) if I lost that. I also use `@package`, since PHP has no build-in concept to enforce architectural boundaries. I'm happy to forbid the rest.

Reply:
Yes! Thanks for reminding me. I did intend to allow (even require) @throws in v3 after discovering that PhpStorm static analysis can only traverse 1-2 levels of exception handling.